Sources/AsyncHTTPClient/HTTPClient+HTTPCookie.swift

```diff
@@ -17,13 +17,16 @@ import NIOHTTP1
 import xlocale
 #elseif canImport(locale_h)
 import locale_h
-#elseif canImport(Darwin)
+#endif
+
+#if canImport(Darwin)
```

Comment:
IMO it would be better to import the other base modules we need like `errno_h`. Darwin in particular is _very_ heavyweight, and if we can avoid it that would be best.